docs/docsite/rst/dev_guide/collection_requirements.rst

```diff
@@ -0,0 +1,437 @@
+
+All of the above can be achieved by using the `GitHub Action template <https://github.com/ansible-collections/collection_template/tree/main/.github/workflows>`_.
+
+To learn how to add tests to your collection, see:
+
+* `Quick-start integration testing guide <https://github.com/ansible/community-docs/blob/main/integration_tests_quick_start_guide.rst>`_
+* `Quick-start unit testing guide <https://github.com/ansible/community-docs/blob/main/unit_tests_quick_start_guide.rst>`_
+
+Collections and Working Groups
+==============================
+
+The collections have:
+
+* Working group page(s) on a corresponding wiki if needed. Makes sense if there is a group of modules for working with one common entity, for example postgresql, zabbix, grafana, and so on.
+* Issue for agenda (or pinboard if there are not regular meetings) as a pinned issue in the repository.
+
+When moving modules between collections
+=======================================
+
+All related entities must be moved/copied including:
+
+* Related plugins and module_utils files (when moving, be sure it is not used by other modules, otherwise copy).
+* CI and unit tests.
+* Corresponding documentation fragments from ``plugins/doc_fragments``.
+
+Also:
+
+* Change ``M()``, examples, ``seealso``, ``extended_documentation_fragments`` to use actual FQCNs in moved content and in other collections that have references to the content.
+* Move all related issues, pull requests, and wiki pages.
+* Look through ``docs/docsite`` directory of `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using the ``grep`` command-line utility) to check if there are examples using the moved modules and plugins to update their FQCNs.
+
+See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ for complete details.
+
+Development conventions
+=======================
+
+Besides all the requirements listed in the `Development conventions <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html>`_, be sure:
+
+* Your modules do not query information using special ``state`` option values like ``get``, ``list``, ``query``, or ``info`` -
+  create new ``_info`` or ``_facts`` modules instead (for more information, refer to the `Developing modules guidelines <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_general.html#creating-an-info-or-a-facts-module>`_)
+* ``check_mode`` is supported in all ``*_info`` and ``*_facts`` modules (for more information, refer to the `Development conventions <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html#following-ansible-conventions>`_)
+
+.. _collection_dependencies:
```

Comment:
```suggestion
.. _collection_dependencies_requirements:
```

Reply:
This is a fix to the CI error about duplicate-label. What that error is saying is that another page in this repository already uses that label, so we have to make this one unique.